tasks/windows.ps1

```diff
@@ -39,6 +39,11 @@ Param(
   $Extension_Request
 )
 
+function Set-SecurityProtocol {
+  # Note: 3072 is the enum value for tls12 to support .NET 4.0
+  [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]3072
```

Comment:
Should probably wrap this in  a try catch   just in case.

Reply:
I think we do want this to error loudly if it fails, but I can wrap and warn/throw a useful message instead?